README.md

```diff
@@ -258,6 +259,26 @@ certificate` errors from the `wsk` CLI.
 If your deployment is not working, check our
 [troubleshooting guide](./docs/troubleshooting.md) for ideas.
 
+# Administering OpenWhisk
+
+[Wskadmin](https://github.com/apache/incubator-openwhisk/tree/master/tools/admin) is the tool to perform various administrative operations against an OpenWhisk deployment.
+
+Since wskadmin requires credentials for direct access to the database (that is not normally accessible to the outside), it is deployed in a pod inside Kubernetes that is configured to access it. You can run `wskadmin` with `kubectl`. You need to know the namespace and the deployment name.
+
+You can then invoke it with:
+
+```
+kubectl -n <namespace> -ti <deployment> -- wskadmin <parameters>
```

Comment:
should be `kubectl -n <namespace> -ti exec <deployment> -- wskadmin <parameters>` ?

Reply:
yeah I fix it